README.md

```diff
@@ -48,7 +48,7 @@ File(Code on macOS) > Preferences > Settings to open your [Settings](https://cod
 
 1. Opening or editing a manifest file (`pom.xml` / `package.json` / `requirements.txt`) scans your application for security vulnerabilities, flag them along with 'quick fixes'.
 
-![ screencast ](images/compAnalysis.gif)
+![ screencast ](images/component-analysis.gif)
```

Comment:
Are these renames are done to avoid breaking the README of the existing release? In that case it would be good to have a dedicated folder with version name instead of just file rename.

e.g.
images/0.1.0/component-analysis.gif